resources/js/comment.js

```diff
@@ -0,0 +1,212 @@
+$(document).ready(function () {
+    var arr = new Array(1000);
+    for (var i = 0; i < 1000; i++) {
+        arr[i] = i;
+    }
+    $.ajaxSetup({
+        headers: {
+            'X-CSRF-TOKEN': $('input[name="_token"]').val(),
+        }
+    });
+
+    var content = '';
+    var count = $('#count').val();
+    var review_id = $('#review_id').val();
+    var msg_submit = $('#msg_submit').val();
+    var msg_delete = $('#msg_delete').val();
+    var msg_delete_confirm = $('#msg_delete_confirm').val();
+
+    comment();
+    deleteComment();
+    updateComment();
+    animate();
+    like_cmt();
+    unlike_cmt();
+
+    function loadComments() {
+        $.ajax({
+            url: '/brs/public/comments',
+            method: 'GET',
+            success: function (response) {
+                var time = new Date();
+                count++;
+                content = `
+                <li class="reviews-single-item" id="li` + count + `">
```

Comment:
```suggestion
                content = `
                <li class="reviews-single-item" id="li ${count} ">
```
xem lại ES6 để viết cho ngắn gọn